worlds/hk/GodhomeData.py

```diff
@@ -0,0 +1,98 @@
+from functools import partial
+
+
+godhome_event_names = ["Godhome_Flower_Quest", "Defeated_Pantheon_5", "GG_Atrium_Roof", "Defeated_Pantheon_1", "Defeated_Pantheon_2", "Defeated_Pantheon_3", "Opened_Pantheon_4", "Defeated_Pantheon_4", "GG_Atrium", "Hit_Pantheon_5_Unlock_Orb", "GG_Workshop", "Can_Damage_Crystal_Guardian", 'Defeated_Any_Soul_Warrior', "Defeated_Colosseum_3", "COMBAT[Radiance]", "COMBAT[Pantheon_1]", "COMBAT[Pantheon_2]", "COMBAT[Pantheon_3]", "COMBAT[Pantheon_4]", "COMBAT[Pantheon_5]", "COMBAT[Colosseum_3]", 'PROFICIENTCOMBAT', 'DIFFICULTSKIPS', "CURSED", 'Warp-Junk_Pit_to_Godhome', 'Bench-Godhome_Atrium', 'Bench-Hall_of_Gods', '*Boss_Geo-Massive_Moss_Charger/', 'Bench-Godhome_Atrium/', 'Bench-Hall_of_Gods/', 'Room_Colosseum_01[left1]/', "GODTUNERUNLOCK", 'WARPSTARTTOBENCH', "GG_Waterways", "Warp-Godhome_to_Junk_Pit", "NAILCOMBAT", "BOSS", "AERIALMINIBOSS", "ANYDASH", "FULLDASH", "SIDESLASH", "MILDCOMBATSKIPS", "SPICYCOMBATSKIPS", "FULLSHADOWDASH", "LEFTSHADOWDASH", "RIGHTSHADOWDASH", "ANYSHADOWDASH", "ANYCLAW"]
```

Comment:
warpstarttobench is always false? the way i read how it was explained it seemed like it should be true (the ap assumption that you can always return to Menu)

will address the other suggestions though

Reply:
https://github.com/homothetyhk/RandomizerMod/blob/master/RandomizerMod/Resources/Logic/macros.json#L11 references INCLUDEBENCHEARPSELECT which is false.

This macro allows warping from start to a bench (rather than from a bench back to start) under certain conditions, for example allowing bench rando to grant you new things in logic by warping to benches granted as items

Reply:
ok it looks like all of the impacted logic were self-referential bench logic (which makes sense with your description) which i removed in a previous commit anyways, and cleaned up the warp to bench event itself just now